api/rbdigital.py

```diff
@@ -1938,34 +1938,34 @@ def invoke(self):
             timestamp.counter += 1
         return result
 
-class OneClickDeltaMonitor(OneClickSyncMonitor):
+class RBDigitalDeltaMonitor(RBDigitalSyncMonitor):
 
-    SERVICE_NAME = "OneClick Delta Sync"
+    SERVICE_NAME = "RBDigital Delta Sync"
 
     def invoke(self):
         return self.api.populate_delta()
 
-class OneClickCirculationMonitor(CollectionMonitor):
-    """Maintain LicensePools for OneClick titles.
+class RBDigitalCirculationMonitor(CollectionMonitor):
+    """Maintain LicensePools for RBDigital titles.
 
     Bibliographic data isn't inserted into new LicensePools until
     we hear from the metadata wrangler.
     """
-    SERVICE_NAME = "OneClick CirculationMonitor"
+    SERVICE_NAME = "RBDigital CirculationMonitor"
```

Comment:
We need a migration script for this change, so that existing installations don't lose track of when they last ran their monitor.

Reply:
Note to posterity: the migration script was added to circulation, since only circulation managers use RBDigital.